hf1.html

```diff
@@ -67,7 +80,27 @@ <h1>Introduction</h1>
 <li>Ivor Grattan-Guinness in mathematics, "Solving Wigner's mystery: The reasonable (though perhaps limited) effectiveness of mathematics in the natural sciences".[9]</li>
 <li>Vela Velupillai in economics, "The Unreasonable Ineffectiveness of Mathematics in Economics".[10]</li>
 </ul>
+
 </article>
-<footer><a id="bottom" href="#top" ><span style="background-color: coral;" title="Jump to top of page">This is the FOOTER AREA</span></a></footer> 
+<hr>
+<form id="cikk_kuldes_ismerosnek" action="hf1.html">
+    <p class="formcim">Cikk küldése ismerősnek</p>
+    <label for="name">Neved*: </label>
+    <input class="urlapmezo" type="text" id="name" name="name" size="100"><br />
+    <label for="tel" pattern="[0-9]{3}-[0-9]{3}-[0-9]{3}">Telefonszámod (xxx-xxx-xxx formában): </label>
```

Comment:
Érdemes bedobni a validátorba, sorol néhány könnyen javítható hibát: https://validator.w3.org/nu/#textarea

Reply:
Done, tx